### PR TITLE
Fixed minor formatting bug in primers/classes

### PR DESCRIPTION
**Pull request by Venkatavaradan-R**
A missing breakline in [primers/classes](https://github.com/chapel-lang/chapel/blob/master/test/release/examples/primers/classes.chpl) was causing a list to render incorrectly on the [website](https://chapel-lang.org/docs/primers/classes.html). 
Fixed that and modified README in [doc/](https://github.com/chapel-lang/chapel/blob/master/doc/README.rst) directory for a bit more clarity as to when the html/ directory is generated.

### CODE REVIEW

**Comment by Venkatavaradan-R** [comment]
@lydia-duncan Please have a look.

File: doc/README.rst [hunk 1]
```diff
@@ -3,9 +3,10 @@ Chapel Documentation
 
 This directory contains Chapel's documentation in two formats:
 
-``html/`` contains a web-rendered format of the documentation, which reflects
-the online documentation hosted at ``https://chapel-lang.org/docs``. This can be
-viewed by pointing a web browser of your choice to ``html/index.html``.
+``html/`` once generated by the ``make docs`` command, contains a web-rendered
```
Comment (lydia-duncan) [inline doc/README.rst:6]: I'd have this say "is generated by the ``make docs`` command and contains" instead

**Review by lydia-duncan — APPROVED** [review]
Other than that minor wording change, I think this is good to go, thanks!  I built the docs and checked that the list was rendered appropriately

**Comment by Venkatavaradan-R** [comment]
I've made the changes :+1:

**Comment by lydia-duncan** [comment]
Thanks!  I'll merge this when master is unfrozen again

**Pull request merged**